flexmeasures/ui/static/css/flexmeasures.css

```diff
@@ -1007,8 +1007,8 @@ body .dataTables_wrapper .dataTables_paginate .paginate_button.current:hover {
     vertical-align: top;
     border-top: 1px solid var(--light-gray) ;
 }
-.left-sidepanel-label{
-    margin-left: 20px;
+.date-margin{
```

Comment:
I guess the `.date-margin` is not necessary, just `!important` is enough?

Reply:
it was for enforcing the margin-left of 20px that i included

Reply:
Yes, but no new CSS class is needed, you could add `!important` right after `margin-left: 20px;` within the `.left-sidepanel-label{` directive, I believe.

Reply:
alright i've sorted